.github/workflows/build.yml

```diff
@@ -13,14 +13,16 @@ jobs:
       matrix:
         redis-version: [5]
 
-        ruby-version: [2.5]
-
-        rspec-core-version: [
-          3.8.2,
-          3.9.3,
-          3.10.1,
-          3.11.0,
-        ]
+        ruby-version:
+          - 3.1
```

Comment:
Why did you remove 2.5 ?

Reply:
I turned the list into the format which is best-loved by the YAML dialect in GitHub Actions. 2.5 is at the end of that list.

Reply:
Damn I am so blind. Sorry.

Reply:
How do _you_ feel about adding 2.6, perhaps 2.7?

Reply:
We don't really care about these version, as we don't use them. That being said, we should add version as needed, if needed.